packages/modelence/src/client/config.ts

```diff
@@ -0,0 +1,37 @@
+declare global {
+  var modelenceConfig: ModelenceConfig;
```

Comment:
Can we call this something else? We already have configs and it does something completely different (i.e. app configs).